modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -507,6 +502,11 @@ public void processOpts() {
             apiTemplateFiles.put("apiDelegate.mustache", "Delegate.java");
         }
 
+        if (apiFirst) {
+            apiTemplateFiles.clear();
```

Comment:
Worth adding a comment why this was moved so someone doesn't move it back later?